woocommerce/woocommerce
# #23641 Issues fix broken links in captions changed photoswipe js options and added html to allow

### All Submissions:






### Changes proposed in this Pull Request:


Before my changes, it was allowing text only in captions, I have changed the js parameters so that now it will allow HTML also and display caption in HTML format if there is an HTML text inserted

Closes #23641.

### How to test the changes in this Pull Request:

1. Add any HTML format text in the image gallery or product featured image. 
2. Goto that product page and click on zoom image and check the caption.
3. Now it will render HTML if HTML is added.

### Other information:




### Changelog entry

> Changed javascript file named "single-product.js" at the below path - assets/js/frontend
